db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/SYSFOREIGNKEYSRowFactory.java

```diff
@@ -365,37 +371,132 @@ String getRefActionAsString(int raRule)
 	}
 
     public static String SYSCAT_REFERENCES_VIEW_SQL = "create view REFERENCES as \n" +
```

Comment:
Given this view is getting to a considerable size, do you mind moving the SQL query to a separate file and loading it statically? For instance putting the sql into `syscat_references_view.sql` and initializing `SYSCAT_REFERENCES_VIEW_SQL` like:
```
static {
   URL url = Resources.getResource("syscat_references_view.sql");
   SYSCAT_REFERENCES_VIEW_SQL = Resources.toString(url, StandardCharsets.UTF_8);
}
```

Reply:
Done.